src/domains/job/recruitment/components/form/WriteRecruitmentPostingForm.jsx

```
@@ -16,7 +16,7 @@ import ToastService from '../../../../../services/toast/ToastService'
 import useFreeDraftStore from '../../../../../store/mypage/useFreeDraftStore.js'
 import { validateExperience } from '../../write/utils/validateExperience'
 
-const WriteRecruitmentPostingForm = ({ onSubmit, editorRef }) => {
+const WriteRecruitmentPostingForm = ({ onSubmit, editorRef, defaultValues }) => {
```

ESLint

[error] 'onSubmit' is missing in props validation

(react/prop-types)